include/net/net_context.h

```diff
@@ -762,7 +762,7 @@ int net_context_send(struct net_pkt *pkt,
 		     void *user_data);
```

Comment:
The PR looks ok, except the commit subject looks a bit weird. Could we say here `net: context: Fix send function documentation` instead, there is no need to talk about function names in the subject as those are mentioned in commit body anyway.

Reply:
and s/void *bug/void *buf

Reply:
> and s/void *bug/void *buf